app/code/Dev/ProductComment/etc/module.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Dev_ProductComment" setup_version="0.0.1"> <sequence>
```

Comment:
Please beautify code. Should be like this
`
<module name="Dev_ProductComment" setup_version="0.0.1">
    <sequence>
        <module name="Magento_Catalog"/>
    </sequence>
</module>
`